frontend/src/styles/_uswds-theme.scss

```diff
@@ -10,7 +10,7 @@ in the form $setting: value,
 @use "uswds-core" with (
   // Disable scary but mostly irrelevant warnings:
   $theme-show-notifications: false,
-
+  $theme-show-compile-warnings: false,
```

Comment:
Necessary to suppress a number of Sass warnings about upcoming versions. We are tied to uswds which is where a lot of them originated.